spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -48,6 +48,16 @@
     DocAuth::LexisNexis::Config.new
   end
 
+  context 'when the dob is incorrectly parsed' do
+    let(:response) { described_class.new(success_response, false, config) }
+    let(:bad_pii) { { dob_year: 'OCR', dob_month: 'failed', dob_day: 'to parse' } }
+
+    it 'does not throw an exception when getting pii from doc' do
+      allow(response).to receive(:pii).and_return(bad_pii)
+      expect { response.pii_from_doc }.not_to raise_error(Date::Error, 'invalid date')
```

Comment:
do we want to not raise `Date::Error`? or not raise any error at all?

```suggestion
      expect { response.pii_from_doc }.not_to raise_error
```

I'd expect no errors at all, so ^^

Reply:
Yup - implemented in 94d8fcd